### ISSUE TITLE
Could we get access to use mat_reloadallmaterials in lua without having to have sv_cheats 1 set?

### ISSUE THREAD

**Issue by VaasKahnGrim** [Facepunch/garrysmod-requests#1563]
Basically the command requires sv_cheats to be set to 1 in order to be used. making it useless in just about all cases

**Comment by robotboy655** [comment]
It's a developer command. Why do you need it when not developing?

**Comment by VaasKahnGrim** [comment]
Because sometimes textures loaded in using game.MountGMA don't want to show up on clients.

**Comment by MyDimeIsUp** [comment]
I agree with @VaasKahnGrim its not 100% for sv_cheats environments, but @VaasKahnGrim you should be using the 2nd argument returned from game.MountGMA which returns all the files in said GMA. Just loop through the files and regex anything with a vtf and mat_reloadtexture that single material, rather than reloading ALL the materials.

But still with @VaasKahnGrim , it shouldn't be limited to developer enviroments

**Comment by thegrb93** [comment]
I think the mountgma issue should rather be fixed.

**Comment by VaasKahnGrim** [comment]
tbh both would be nice if fixed. MountGMA because then you wouldn't have to bother with concommands, and lua bindings for mat_reloadtexture stuff for dealing with textures and stuff
